aemedge/blocks/columns/columns.js

```diff
@@ -15,4 +103,10 @@ export default function decorate(block) {
       }
     });
   });
+
+  findAndMarkFragmentLinks(block);
```

Comment:
Can we avoid calling these methods outside `decorate` function ?

Reply:
What do you mean? They are called from the decorate function.

Reply:
My bad, I didn't saw last bracket ;)